python_ta/cfg/cfg_generator.py

```diff
@@ -13,6 +13,7 @@
 from astroid import nodes
 from astroid.builder import AstroidBuilder
 
+from ..transforms.z3_visitor import Z3Visitor
```

Comment:
@Raine-Yang-UofT I'm not sure how you tried to test what I asked for, but `transforms.z3_visitor` imports `z3`, and so will raise an `ImportError` if `z3` isn't installed.